### DISCUSSION TITLE
Notifications

### DISCUSSION THREAD

**Discussion by Vinc0110** [scikit-rf/scikit-rf#773] [Q&A] [answered]
The discussions feature is relatively new and I was missing most of the activity here, because I don't remember to check this section when I read issues or PRs. Also, there is no number next to the "Discussions" tab unlike for the "Issues" and "Pull requests" tab to indicate the activity.

How do you do it? Are you just better at remembering to go here and check or can I activate email notifications somewhere?

**Comment by denzchoe** [comment, marked as answer]
hmm I received notifications of #773 here in my email inbox @Vinc0110

**Reply by jhillairet** [reply]
I'm also receiving emails like for usual issue or PR.

**Reply by Vinc0110** [reply]
Ok, seems like my global email settings for GitHub are the problem.
![image](https://user-images.githubusercontent.com/63644854/196784747-a1db0c41-ab10-4151-a6c5-5c42a52cb308.png)
